ExternalBeamPlanning/Widgets/CMakeLists.txt

```diff
@@ -42,8 +68,22 @@ set(${KIT}_MOC_SRCS
   qSlicerDoseEngineLogic.h
   qSlicerMockDoseEngine.h
   qSlicerScriptedDoseEngine.h
+  qSlicerAbstractPlanOptimizer.h
+  qSlicerPlanOptimizerPluginHandler.h
+  qSlicerPlanOptimizerLogic.h

```

Comment:
It is strange to have classes called Logic in the Widgets project. I'm sure there is a good reason, let's discuss this later.